### PR TITLE
SDCORE-859: Review of smf changes to handle PDU Session Release

### PR DESCRIPTION
**Pull request by UpadhyayaAnkur**

### CODE REVIEW

File: producer/n1n2_data_handler.go [hunk 1]
```diff
@@ -101,7 +101,7 @@ func HandleUpdateN1Msg(txn *transaction.Transaction, response *models.UpdateSmCo
 			smContext.ChangeState(smf_context.SmStateInit)
 			smContext.SubCtxLog.Traceln("PDUSessionSMContextUpdate, SMContextState Change State: ", smContext.SMContextState.String())
 			response.JsonData.UpCnxState = models.UpCnxState_DEACTIVATED
-			/*problemDetails, err := consumer.SendSMContextStatusNotification(smContext.SmStatusNotifyUri)
+			problemDetails, err := consumer.SendSMContextStatusNotification(smContext.SmStatusNotifyUri)
```
Comment (thakurajayL) [inline producer/n1n2_data_handler.go:104]: Can you please add intel copyright ?

Comment (UpadhyayaAnkur) [reply producer/n1n2_data_handler.go:104]: Done

File: producer/n1n2_data_handler.go [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // SPDX-FileCopyrightText: 2021 Open Networking Foundation <info@opennetworking.org>
 // Copyright 2019 free5GC.org
-//
+// SPDX-FileCopyrightText: 2022-present Intel Corporation
```
Comment (thakurajayL) [inline producer/n1n2_data_handler.go:3]: Keep the latest on top..

**Comment by UpadhyayaAnkur** [comment]
retest this please

**Comment by UpadhyayaAnkur** [comment]
retest this please

**Pull request closed without merging**